inlong-manager/manager-dao/src/main/resources/mappers/SortIdParamsEntityMapper.xml

```diff
@@ -0,0 +1,121 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+
+<!DOCTYPE mapper PUBLIC "-//mybatis.org//DTD Mapper 3.0//EN" "http://mybatis.org/dtd/mybatis-3-mapper.dtd">
+<mapper namespace="org.apache.inlong.manager.dao.mapper.SortIdParamsEntityMapper">
+  <resultMap id="BaseResultMap" type="org.apache.inlong.manager.dao.entity.SortIdParamsEntity">
+    <id column="id" jdbcType="INTEGER" property="id" />
+    <result column="task_name" jdbcType="VARCHAR" property="taskName" />
+    <result column="type" jdbcType="VARCHAR" property="type" />
+    <result column="group_id" jdbcType="VARCHAR" property="groupId" />
+    <result column="stream_id" jdbcType="VARCHAR" property="streamId" />
+  </resultMap>
+  <sql id="Base_Column_List">
+    id, type, group_id, stream_id
+  </sql>
+  <sql id="Base_id_params">
+    task_name, type, group_id, stream_id
+  </sql>
+  <select id="selectByPrimaryKey" parameterType="java.lang.Integer" resultMap="BaseResultMap">
+    select 
+    <include refid="Base_Column_List" />
+    from sort_id_params
+    where id = #{id,jdbcType=INTEGER}
+  </select>
+  <delete id="deleteByPrimaryKey" parameterType="java.lang.Integer">
+    delete from sort_id_params
+    where id = #{id,jdbcType=INTEGER}
+  </delete>
+  <insert id="insert" parameterType="org.apache.inlong.manager.dao.entity.SortIdParamsEntity">
+    insert into sort_id_params (id, task_name, type, 
+      group_id, stream_id)
+    values (#{id,jdbcType=INTEGER}, #{taskName,jdbcType=VARCHAR}, #{type,jdbcType=VARCHAR}, 
+      #{groupId,jdbcType=VARCHAR}, #{streamId,jdbcType=VARCHAR})
+  </insert>
+  <insert id="insertSelective" parameterType="org.apache.inlong.manager.dao.entity.SortIdParamsEntity">
+    insert into sort_id_params
+    <trim prefix="(" suffix=")" suffixOverrides=",">
+      <if test="id != null">
```

Comment:
Shouldn't Id be generated automatically here? Why should assign an ID value when inserting?

Reply:
These codes are auto-generated by mybatis generator plugins. When insert one record, the feild "id" should not be assign.